ros2_easy_test/ros2_easy_test/env.py

```diff
@@ -363,3 +374,132 @@ def call_service(
+
+        with self._registered_actions_lock:
+            try:
+                return self._registered_actions[name]
+            except KeyError:
+                # This is a bit tricky but relieves the user from passing it explicitly
+                module = import_module(goal_class.__module__)
+                # We cut away the trailing "_Goal" from the type name, which has length 5
+                base_type_name = goal_class.__name__[:-5]
+                base_type: Type = getattr(module, base_type_name)
+
+                client = ActionClient(self, base_type, name, callback_group=self._action_cb_group)
+                self._registered_actions[name] = client
+                return client
+
+    def send_action_goal(
+        self,
+        name: str,
+        goal: Any,
+        collect_feedback: bool = True,
+        timeout_availability: Optional[float] = 1,
+        timeout_receive_goal: Optional[float] = 1,
+    ) -> Tuple[ClientGoalHandle, List[Any]]:
+        """Sends the goal to the given action but does not handle the result.
+
+        This does not check if the goal was accepted and does not attempt to retrieve the result.
+
+        Args:
+            name: The name of the action
+            goal_msg: Goal message to send to the action server.
+            collect_feedback: Whether to collect feedbacks asynchronously. Defaults to True.
+                This is provided for performance reasons.
+            timeout_availability: The timeout to wait for the action to be available. Defaults to 1.
+            timeout_accept_goal: The timeout to wait for the action server to accept the goal. Defaults to 1.
+
+        Raises:
+            TimeoutError: If the action server does not respond within the specified timeouts.
+
+        Returns:
+            Return the goal handle and feedback list (that might still grow asynchronously).
+
+        See Also:
+            :meth:`send_action_goal_and_wait_for_result`
+        """
+
+        # Create an action client
+        client = self._get_action_client(name, type(goal))
+
+        # Wait for the action to be available
+        if not client.wait_for_server(timeout_availability):
+            raise TimeoutError(f"Action {name} did not become ready within {timeout_availability} seconds")
+
+        # Prepare collecting feedbacks
+        feedbacks: List[RosMessage] = []
+
+        def feedback_callback(feedback_msg: RosMessage) -> None:
+            feedbacks.append(feedback_msg.feedback)
+
+        # Send the goal and wait for the acceptence
+        send_goal_future = client.send_goal_async(
+            goal, feedback_callback=feedback_callback if collect_feedback else None
+        )
+        goal_handle: ClientGoalHandle = self.await_future(send_goal_future, timeout=timeout_receive_goal)
+
+        # Return the results to the test case
+        return goal_handle, feedbacks
+
+    def send_action_goal_and_wait_for_result(
```

Comment:
One intermediate step might be handy?

`send_action_goal_and_wait_for_acceptance`

Reply:
Yeah, but I don't think having a method essentially calling another one and doing `assert goal_handle.accepted, f"Goal was not accepted: {goal_handle.status=}"` is worth having. It would be too many lines and a new method name for only a minor convenience.